benchmarks/mx_formats/cast_bench.py

```diff
@@ -4,18 +4,26 @@
 # This source code is licensed under the BSD 3-Clause license found in the
 # LICENSE file in the root directory of this source tree.
 
-from typing import Callable, Tuple
+from functools import partial
+from typing import Tuple
 
 import fire
 import torch
 import triton
-from torch._inductor.utils import do_bench_using_profiling
+from triton.testing import do_bench
 
 from torchao.prototype.mx_formats.kernels import (
     triton_to_mxfp8_dim1,
 )
 from torchao.prototype.mx_formats.mx_tensor import to_mx
 
+try:
+    import mxfp8_cuda
```

Comment:
what would it take to just have this available in torchao instead of requiring a separate import?

Reply:
Migrated to torchao/csrc/cuda/mx_kernels and updated setup.py to make it available as `torchao.prototype.mxfp8_cuda`. cc @drisspg as well